visualizer/info-box.js

```diff
@@ -44,7 +44,13 @@ class InfoBox extends HtmlContent {
     }
 
     this.functionText = node.functionName
+
     this.pathText = node.fileName
```

Comment:
Very picky 😳, but I suppose since this is not an HTML string it might be better to rename this property to `pathHtml` or `pathHTML`. Whichever way is more natural 👍